src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -37,19 +39,125 @@
  * @author Dragan Bozanovic (bozanovicdr@gmail.com)
  * @version $Id$
  * @since 0.25
- * @todo #377:30min/DEV We need more tests for TkProxy.
+ *
  *  The tests should verify the different HTTP methods (GET, POST, etc),
  *  as well as the different combinations of request/response headers.
  */
 public final class TkProxyTest {
 
     /**
+     * Test string.
+     */
+    public static final String HELLO_WORLD = "hello, world!";
```

Comment:
@aschworer First of all, this field should NOT be public.
In this particular case, this field can be easily removed - simply use local variables (different) in each test.